app/views/finders/_radio_facet.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="facet-container">
+<div class="facet-container" data-ga4-change-category="update-filter radio" data-ga4-section="">
```

Comment:
This is intentional, this is what our PA told me. Initially they thought the "Topic" section above the radio buttons on https://www.gov.uk/official-documents was the `section` value, but it isn't (Topic is its own accordion), so they said:

>  I think an empty string might make the most sense because there is nothing there to pass (but we did want to pass a value so i don’t think undefined quite makes sense either)

Reply:
Okay, makes sense.